### DISCUSSION TITLE
to many Deads

### DISCUSSION THREAD

**Discussion by Jok3r89** [swar/Swar-Chia-Plot-Manager#1083] [General]
Hi guys, 
i m using swar since two mount on two machines and have trouble with deads on one.

#### System
CPU Ryzen 9 5950x OC 4,4Ghz (all core)
64Gb DDR4 OC 3766mHz
2x MVNE Samsung 980 pro
2x HP Enterprise SSDs U.2 with pci x4 connector.
Asus B550 gaming e

#### Settings 4x ->

– name: P1
max_plots: 999
farmer_public_key:
pool_public_key:
temporary_directory: G:\
temporary2_directory:
destination_directory: H:\
size: 32
bitfield: true
threads: 6
buckets: 128
memory_buffer: 8000
max_concurrent: 3
max_concurrent_with_start_early: 4
initial_delay_minutes: 0
stagger_minutes: 110
max_for_phase_1: 2
concurrency_start_early_phase: 4
concurrency_start_early_phase_delay: 0
temporary2_destination_sync: false
exclude_final_directory: false
skip_full_destinations: true
unix_process_priority: 10
windows_process_priority: 128
enable_cpu_affinity: false
cpu_affinity: [ 0, 1, 2, 3, 4, 5 ]
.........

Has anyone an idea about my issue ? 
What are the reasons for deads?

Thanks in adv.
Dominik

**Comment by w4d3j** [comment]
Are you implying that you have 3 other jobs with the same options?  So 4 total?  I'll run with that.

With 4 jobs @ those settings, you're telling the program it may use 72 threads (6*3=18 per job) and 96 GB of RAM (8000x3=24 GB per plot).  Then there's the Start Early plots, making it 96 threads and 128GB RAM.  You do not have this many threads or this much RAM.  

And some thoughts:

- 6 threads is way too many for a job.  Unless you're using a customized chiapos, each job only uses multiple threads in Phase 1.  And after 3-4 threads, the speed gains become marginal.
- 8000MB is way too much RAM.  You won't see any performance increase over ~4000.
